tests/cypress/integration/issue_1599_pl_user_registration.js

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
The same comments like in the previous file

Reply:
Done.